cpu/native/startup.c

```diff
@@ -577,6 +586,10 @@ __attribute__((constructor)) static void startup(int argc, char **argv, char **e
     periph_init();
     board_init();
 
+    /* should not error as SIGUSR is a valid error code and EINVAL is the only
+     * specified error code => don't check return value */
+    signal(SIGUSR1, reset_handler);
```

Comment:
I think you need to (should) use: https://github.com/RIOT-OS/RIOT/blob/master/cpu/native/irq_cpu.c#L412 instead

Reply:
or additionally?

Reply:
Might be.